features/invitelist/impl/src/test/kotlin/io/element/android/features/invitelist/impl/InviteListPresenterTests.kt

```diff
@@ -500,4 +492,16 @@ class InviteListPresenterTests {
             unreadNotificationCount = 0,
         )
     )
+
+    private fun aPresenter(
```

Comment:
Just a remark about the naming, I think we reserve `a.*()` name for functions that create fixtures. I think `createPresenter` would be more appropriate since this is the class we are tested here.